tests/snapshots/compile_long__huge_tree.bend.snap

```diff
@@ -2,4 +2,6 @@
 source: tests/golden_tests.rs
 input_file: tests/golden_tests/compile_long/huge_tree.bend
 ---
-Compiled
+[4m[1m[31mErrors:[0m
+[1mIn definition '[4mmain[0m[1m':[0m
+  Definition is too large for hvm (size=120002, max size=4095). Please break it into smaller pieces.
```

Comment:
This is weird, it should be breaking this thing in pieces of the size we passed to the `float_combinators` pass (64, MAX_NET_SIZE_CUDA)

Reply:
I'll have a look into why this is later